### PR TITLE
chore(cli): request_vc print valid json string

### PR DESCRIPTION
**Pull request by jonalvarezz**
The CLI's request_vc command was printing invalid json strings. This change fixes it.

[Before](https://sharetext.me/dbuecjq01h) vs [now](https://sharetext.me/j0mziwnvft)

### CODE REVIEW

File: tee-worker/cli/src/trusted_base_cli/commands/litentry/request_vc.rs [hunk 1]
```diff
@@ -463,9 +462,9 @@ impl RequestVcCommand {
 		match perform_trusted_operation::<RequestVCResult>(cli, trusted_cli, &top) {
 			Ok(mut vc) => {
 				let decrypted = aes_decrypt(&key, &mut vc.vc_payload).unwrap();
-				let credential: Credential = serde_json::from_slice(&decrypted).unwrap();
+				let credential_str = String::from_utf8(decrypted).expect("Found invalid UTF-8");
```
Comment (jonalvarezz) [inline tee-worker/cli/src/trusted_base_cli/commands/litentry/request_vc.rs:463]: vc_payload is already an encoded json string, so I skipped the `serde_json` parsing

**Review by Kailai-Wang — COMMENTED** [review]
Thanks!

Can you please adjust `request_vc_direct.rs` too? :)

**Review by BillyWooo — APPROVED** [review]

**Comment by jonalvarezz** [comment]
> Thanks!
> 
> Can you please adjust `request_vc_direct.rs` too? :)

done!

**Review by Kailai-Wang — APPROVED** [review]
Nice!

**Pull request merged**